socorro/scripts/pubsub.py

```diff
@@ -134,18 +134,36 @@ def status(config, args):
                 print('             %s' % crash_id)
 
 
-def publish_crashid(config, args):
+def publish_crashid(config, queue, crashids):
     publisher = pubsub_v1.PublisherClient()
     project_id = config['resource.pubsub.project_id']
 
-    queue = args.queue
     topic_name = config['resource.pubsub.%s_topic_name' % queue]
     topic_path = publisher.topic_path(project_id, topic_name)
 
-    for crash_id in args.crashid:
+    for crash_id in crashids:
         future = publisher.publish(topic_path, data=crash_id.encode('utf-8'))
         future.result()
         print('Published: %s' % crash_id)
+    else:
+        print('No crash ids provided.')
```

Comment:
Oops--I should have caught this in review. This clause executes if `for` gets through the iterable without breaking out. That happens whether or not there were crashids to publish.

I think what we want is something like this:

```
if crashids:
    for crash_id in crashids:
        blahblahblah
else:
    print('No crash ids provided.')
```

Or alternatively:

```
if not crashids:
    print('No crash ids provided. I am sad.')
    return

for crash_id in crashids:
    blahblahblah()
```

Reply:
oh no I've learned ``else`` wrong. 🙈 

I think I can drop the clause entirely, since the detection for an empty list is done by the caller.